views/agency-requests/edit_participant.php

```diff
@@ -0,0 +1,118 @@
+<?php
+# Get people id from params
+$peopleid = rawurldecode(implode('/', $params));
+
+
+$result = $db->query("SELECT participants.participantid, participants.dateofbirth, participants.race, people.firstname, people.lastname, people.middleinit " .
+					"FROM participants " .
+					"INNER JOIN people ON participants.participantid = people.peopleid WHERE people.peopleid=$1", [$peopleid]);
+
+$participant = pg_fetch_assoc($result);
+
+$phoneType='Primary';
+$phoneInfo = $db->query("SELECT * from people INNER JOIN participants ON people.peopleid = participants.participantid
+INNER JOIN forms ON participants.participantid = forms.participantid
+INNER JOIN formphonenumbers ON forms.formid = formphonenumbers.formid WHERE peopleid = $1 AND phonetype = $2 ", [$peopleid, $phoneType]);
+
+
+$primaryPhone = pg_fetch_assoc($phoneInfo);
+
+$phoneType='Secondary';
+$phoneInfo = $db->query("SELECT * from people INNER JOIN participants ON people.peopleid = participants.participantid
+INNER JOIN forms ON participants.participantid = forms.participantid
+INNER JOIN formphonenumbers ON forms.formid = formphonenumbers.formid WHERE peopleid = $1 AND phonetype = $2 ", [$peopleid, $phoneType]);
+
+
+$secondaryPhone = pg_fetch_assoc($phoneInfo);
+
+$formid= $secondaryPhone['formid'];
+
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST'){
+	$fname = $mname = $lname = null;
+	
+	if(isset($_POST["fname-update"])){
+		$fname = $_POST["fname-update"];
+		$update = $db->query("UPDATE people SET firstname = $1 WHERE peopleid = $2",[$fname, $peopleid]);
+		
+	}
+	
+	if(isset($_POST["mname-update"])){
+		$mname = $_POST["mname-update"];
+		$update = $db->query("UPDATE people SET middleinit = $1 WHERE peopleid = $2",[$mname, $peopleid]);
+	}
+	if(isset($_POST["lname-update"])){
+		$lname = $_POST["lname-update"];
+		$update = $db->query("UPDATE people SET lastname = $1 WHERE peopleid = $2",[$lname, $peopleid]);
+	}
+	if(isset($_POST["pphone-update"])){
+		$primary = $_POST["pphone-update"];
+		echo $primary;
+		$update = $db->query("UPDATE formphonenumbers SET phonenumber = $1 WHERE formid = $2",[$primary , $formid]);
+	}
+	if(isset($_POST["sphone-update"])){
+		$secondary = $_POST["sphone-update"];
+		echo $secondary;
+		$update = $db->query("UPDATE formphonenumbers SET phonenumber = $1 WHERE formid = $2", [$secondary , $formid]);
+	}
+	echo "<div class=\"alert alert-outline-success w-100\" style=\"display:block;\" role=\"alert\">
+			Participant information has been updated
+			</div>
+			";
+	
+$result = $db->query("SELECT participants.participantid, participants.dateofbirth, participants.race, people.firstname, people.lastname, people.middleinit " .
+					"FROM participants " .
+					"INNER JOIN people ON participants.participantid = people.peopleid WHERE people.peopleid=$1", [$peopleid]);
+
+$participant = pg_fetch_assoc($result);
+	
+}
+
+
+?>
+
+    <a href="/back"><button class="cpca btn"><i class="fa fa-arrow-left"></i> Back</button></a>
```

Comment:
`/back` route doesn't exist any more. Either use a specific route or the `goBack()` function (which is by no means perfect and needs to be fixed anyways haha)